ci/solana-testnet-prebuilt.yml

```diff
@@ -0,0 +1,13 @@
+steps:
```

Comment:
let's merge this with solana-testnet.yml, but with:

```diff
-command: "ci/snap.sh"
+command: "test -n $PREBUILT || ci/snap.sh"
```

Reply:
👍